applications/h264/h264_endoscopy_tool_tracking/README.md

```diff
@@ -25,35 +25,29 @@ can be specified in the 'h264_endoscopy_tool_tracking.yaml' file.
 
 The data is automatically downloaded when building the application.
 
-## Building And Running H.264 Endoscopy Tool Tracking Application
+## Building and Running H.264 Endoscopy Tool Tracking Application
 
-Follow steps in README.md from parents directory to build and run the Holohub
-dev container. Once inside the Holohub dev container, follow steps mentioned
-below to build and run H.264 Endoscopy Tool Tracking application.
-
-## Building the application
-
-Once inside Holohub dev container, run below command from a top level Holohub
-directory.
+* Building and running the application from the top level Holohub directory:
 
 ```bash
-./run build h264_endoscopy_tool_tracking
+# C++ version
+./dev_container build_and_run h264_endoscopy_tool_tracking --docker_file applications/h264/Dockerfile --language cpp
+
+# Python version
+./dev_container build_and_run h264_endoscopy_tool_tracking --docker_file applications/h264/Dockerfile --language python
 ```
 
-## Running the application
+Important: on aarch64, applications also need tegra folder mounted inside the container and
+the `LD_LIBRARY_PATH` environment variable should be updated to include
+tegra folder path.
 
-* Running the application from the top level Holohub directory
+Open and edit the [Dockerfile](../Dockerfile) and uncomment line 66:
 
 ```bash
-./run launch h264_endoscopy_tool_tracking
+# Uncomment the following line for aarch64 support
```

Comment:
I think we can later find a better way to handle this.

Reply:
Agree. Or we can always include it? It won't hurt even when the path doesn't exist.